### PR TITLE
fix: time standardization in two formats only

### LINKED ISSUE #5490: Date and Time: Standardize it across system
The date and time are using different formats in the system. Please identify different places for the date and time usage and standardize the date and time in all ways.

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5490
Make event format as in this example: 4 Dec, 2020 5:00 PM (IST)
 - [x] table https://eventyay.com/events
 - [x] table https://eventyay.com/admin/events
 - [x] table https://eventyay.com/admin/sales
 - [x] table https://eventyay.com//admin/users/
- [x]  table event orders table /events/[event ID]/tickets/orders
- [x]  table event attebdees /events/[event ID]/tickets/attebdees
 - [x]  table session overview page events/[event ID]/sessions/all
This is good for most public pages
- [x]  Make event format as in this example: Friday, 4 December, 2020 5:00 PM (IST)
 - [x] tickets overview https://eventyay.com/my-tickets/past
 - [x] ticket order https://eventyay.com/orders/[order number here]
 - [x] events dashboard /events/[events ID]
- [x]  public events page header e/[eventpage], e.g. https://eventyay.com/e/a315169f
- [x]  public events page ticket section, e.g. https://eventyay.com/e/a315169f#tickets
 
 call for speakers, e.g. https://eventyay.com/e/a315169f/cfs
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Please also change using long version on:
* frontpage event tiles
* event pages above calendar icon

**Comment by mariobehling** [comment]
Looks like you are changing the time format to 24 hours and show e.g. 13:00 PM. Please use standard time format and we need to take care that this is different in different languages.
Compare: https://open-event-frontend-git-time-standard-format.eventyay.vercel.app/e/3dbaaa50
See PR here as well https://github.com/fossasia/open-event-frontend/pull/5748

**Comment by iamareebjamal** [comment]
Yes, there are far fewer changes in this PR than the previous one

File: app/templates/public.hbs [hunk 1]
```diff
@@ -9,9 +9,9 @@
             <img src="{{this.model.logoUrl}}" class="logo mr-8" />
           {{/if}}
           <div>
-            <h4 class="event time">{{header-date this.model.startsAt this.model.timezone}}</h4>
```
Comment (daretobedifferent18) [inline app/templates/public.hbs:12]: why changed to general

File: app/controllers/events/view/tickets/orders/list.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         cellComponent   : 'ui-table/cell/events/view/tickets/orders/cell-date',
         headerComponent : 'tables/headers/sort',
         width           : 100,
-        dateFormat      : 'MMMM DD, YYYY - HH:mm A',
```
Comment (daretobedifferent18) [inline app/controllers/events/view/tickets/orders/list.js:38]: Pls also correct this declaration, here I think we need to define dateFormat inside options

File: app/templates/public.hbs (hunk 1 above)
Comment (iamareebjamal) [reply app/templates/public.hbs:12]: We want to move to a single helper, general-date

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**